HW8/index.js

```diff
@@ -0,0 +1,46 @@
+const userBirthDate = Number(prompt("What is your year of birth?"))
+const userCity = prompt("City where you were born")
+const userFavSport = prompt("What is your favourite sport?")
+
+
+if (isNaN(userBirthDate)) {
+    userAge = "Wrong number"
+} else if (userBirthDate === 0) {
+    userAge = "It is a pity that you did not want to enter your year of birth"
+} else {
+    userAge = 2022 - userBirthDate
+}
+
+
+if (userCity === null || userCity === "") {
```

Comment:
You can use shorter check
```suggestion
if (!userCity) {
```